src/renderer/components/MapEditor/index.js

```diff
@@ -126,7 +126,9 @@ const MapEditor = () => {
       var saved = history.toJSON()
       id.current.flush()
       if (saved) history.fromJSON(saved)
-      ipcRenderer.send('zoom-to-data-get-centroid', 'node', zoomToData)
+      api.zoomToData('node', (_, loc) => {
```

Comment:
This is now throwing an error because you can't send functions as a callback to `ipcRenderer.send` anymore, but we have the same function available in `ipc`